assets/stylesheets/index.scss

```diff
@@ -6,16 +6,19 @@
 // Base
 @import 'base/button';
 @import 'base/form';
-@import 'base/nav';
 @import 'base/table';
 @import 'base/typography';
 
 // Components
 @import 'components/alert';
+@import 'components/list-search-result';
+@import 'components/search-result';
 
 // Layouts
 @import 'layouts/application';
 @import 'layouts/authentication';
 
 // Screens
-@import 'screens/keyword.scss';
+@import 'screens/keyword-result';
```

Comment:
Good catch. Fixed in 2094674afe24e1f6e6bd1e01c0bc1a7ac7a32460

Reply:
The fix still does not show though 🤔 

![image](https://user-images.githubusercontent.com/696529/105324104-8cae0380-5bfd-11eb-95d5-a20bb7632531.png)

This would apply to all imported stylesheets 🙈 So maybe doing this clean up in a chore task can be the good call here.

Reply:
Created issue for this change in https://github.com/gutakk/go-google-scraper/issues/66